WinFormsUI/MainForm.cs

```diff
@@ -1,5 +1,7 @@
 ﻿using System;
+using System.Collections.Generic;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=4966361)